packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -565,7 +553,7 @@ class NetworkRequestOverviewView extends StatelessWidget {
     );
   }
 
-  Widget _buildHttpTimeGraph(BuildContext context) {
+  Widget _buildHttpTimeGraph() {
```

Comment:
we should move these to private helper widgets instead of helper methods

Reply:
Leaving that for a follow-up enabling the DCM lint to use private helper widgets after enabling the associated lint.